package.json

```
@@ -15,7 +15,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint ."
```

Linter should check only src directory eg.:
```suggestion
    "lint": "eslint 'src/**/*.{js,jsx}'"
```